book/mars_maps.ipynb

```diff
@@ -0,0 +1,188 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "19f4dff3",
+   "metadata": {},
+   "source": [
+    "# Making some Mars maps with pygmt\n",
+    "\n",
+    "This tutorial page covers the basics of creating some maps and 3D plot of Mars (yes! Mars). The idea here is to demonstrate that you can use a simple sequence of commands with PyGMT, a Python wrapper for the Generic Mapping Tools (GMT), and with some public data about the topography of Mars, create your own maps, as well as compare this topography with what we know of our own planet Earth."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b48dd300",
+   "metadata": {},
+   "source": [
+    "## Mars dataset\n",
+    "\n",
+    "First, we open the `mola32.nc` file using xarray. Note the longitudes are from 0-360°, latitudes are distributed from North to South and the `alt`variable is the MOLA Topography at 32 pixels/degree built from original MOLA file `megt90n000fb.img`. The source is the Mars Climate Database from the European Space Agency."
+   ]
+  },
+  {
```

Comment:
Suggest including this cell in both notebooks so that they are independent (i.e., data download step is included in both)
```suggestion
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2f913d8d",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Also, if you are in colab or trying from your jupyter, you will need the Mars Topography (MOLA) already in Netcdf\n",
    "# a copy of the original file distributed from the Mars Climate Database,\n",
    "# from the European Space Agency under ESTEC contract 11369/95/NL/JG(SC) and Centre National D'Etude Spatial\n",
    "# is in the gdrive.\n",
    "\n",
    "!gdown 1fDzz8AxR1T58y0IGPhmbb1ZwrTLckp2G"
   ]
  },
  {
```